Readability.js

```diff
@@ -1370,7 +1460,8 @@ Readability.prototype = {
                        values["twitter:description"];
 
     // get site name
-    metadata.siteName = values["og:site_name"];
+    metadata.siteName = jsonld.siteName ||
+                        values["og:site_name"];
 
     // in many sites the meta value is escaped with HTML entities,
     // so here we need to unescape it
```

Comment:
Security: Does using the JSON-LD values directly, and/or unescaping their HMTL entities cause any security concern in the Firefox integration?

Reply:
I think right now, we expect all these values to be plaintext. Although there won't be a security issue if they include HTML, it'll look bad because we assign to `textContent`. I *think* this is correct for all the existing uses of metadata; if JSON-LD is the "odd one out", we should probably strip the markup out?

Reply:
In my understanding of the threat model here, running `unescapeHTMLEntities` on the plain-text metadata can still generate HTML tags, so being added as `textContent` is the only line of defense here. Since the JSON-LD is not expected to include tags (except for malicious intent), I don't think anything else is strictly needed here, right?